testing/e2e/app/routes/SideNavigation/spec.ts

```diff
@@ -0,0 +1,52 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('SideNavigation tooltips', () => {
+  test('should show tooltip when collapsed', async ({ page }) => {
+    await page.goto('/SideNavigation');
+
+    // hover Home
+    {
+      const homeButton = page.getByRole('button', { name: 'Home' });
+      const tooltip = page.getByText('Home').locator('visible=true');
+      await expect(tooltip).not.toBeVisible();
+      await homeButton.hover();
+      await expect(tooltip).toBeVisible();
+    }
+
+    // hover Settings
+    {
+      const settingsButton = page.getByRole('button', { name: 'Settings' });
+      const tooltip = page.getByText('Settings').locator('visible=true');
+      await expect(tooltip).not.toBeVisible();
+      await settingsButton.hover();
+      await expect(tooltip).toBeVisible();
+    }
+  });
+
+  test('should not show tooltip when expanded', async ({ page }) => {
```

Comment:
I noticed while testing out this PR that the expand button has a tooltip when the side navigation is expanded, and the content buttons do not. Is this intended behavior?

https://github.com/iTwin/iTwinUI/assets/141063311/9f52e568-e0f2-495f-a896-8b4e47bbfb61

Reply:
Yes, this behavior is the same as before. The reason for hiding the tooltips for the main `SidenavButton` is that the label is already visible when expanded, making the tooltip redundant; the expand button does not have a visible label.